components/style/stylist.rs

```diff
@@ -80,7 +80,7 @@ pub struct Stylist {
     viewport_constraints: Option<ViewportConstraints>,
 
     /// If true, the quirks-mode stylesheet is applied.
-    quirks_mode: QuirksMode,
+    pub quirks_mode: QuirksMode,
```

Comment:
If you have the time, while you're here, could you make an accessor instead of just making the field public?